rotkehlchen/api/v1/schemas.py

```diff
@@ -977,15 +974,13 @@ class ExchangesResourceRemoveSchema(Schema):
     location = LocationField(limit_to=SUPPORTED_EXCHANGES, required=True)
 
 
-class ExchangeBalanceQuerySchema(Schema):
+class ExchangeBalanceQuerySchema(AsyncQueryArgumentSchema):
```

Comment:
Same as above here.

This is used here: https://github.com/rotki/rotki/blob/98f49cd3ed26c641fec03b78eff9fe1872385fbf/rotkehlchen/api/v1/resources.py#L441

You will need to change it:

```diff
-    def get(self, location: Optional[Location], async_query: bool, ignore_cache: bool) -> Response:
+    def get(self, async_query: bool, location: Optional[Location], ignore_cache: bool) -> Response:
```

Pretty weird no tests failed. Can you check why also? We may need to have this path tested properly